src/EprPrnIntegration.Common/Configuration/ServiceBusConfiguration.cs.orig

```diff
@@ -0,0 +1,19 @@
+﻿using System.Diagnostics.CodeAnalysis;
```

Comment:
we dont want to have this

Reply:
This file is included by accident, I have removed this